sky/cli.py

```diff
@@ -923,7 +972,10 @@ def launch(
 
 
 @cli.command(cls=_DocumentedCodeCommand)
-@click.argument('cluster', required=True, type=str)
+@click.argument('cluster',
+                required=True,
+                type=str,
+                shell_complete=_complete_cluster_name)
```

Comment:
nit: It would be nice to have this `shell_complete` for sky launch as well, since sometimes the user want to rerun the `sky launch` for setup and file mounts.

Reply:
To clarify, this is for the `--cluster` option for sky launch?